gabbi/handlers/jsonhandler.py

```diff
@@ -86,6 +88,13 @@ def action(self, test, path, value=None):
         except ValueError:
             raise AssertionError('json path %s cannot match %s' %
                                  (path, test.response_data))
+
+        # read data from disk if the value starts with '<@'
+        if isinstance(value, str) and value.startswith('<@'):
+            info = test.load_data_file(value.replace('<@', '', 1))
```

Comment:
I don't like using slices like that, it makes me have to decode the slice grammar whereas what I've got above is visible and readable.